client/browser/package.json

```diff
@@ -22,9 +22,9 @@
     "release:chrome": "yarn run -T webstore upload --auto-publish --source build/bundles/chrome-bundle.zip --extension-id dgjhfomjieaadpoljlnidmbgkdffpack --client-id $GOOGLE_CLIENT_ID --client-secret $GOOGLE_CLIENT_SECRET --refresh-token $GOOGLE_REFRESH_TOKEN",
     "release:firefox": "./scripts/release-firefox.sh",
     "release:npm": "TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\":\\\"commonjs\\\"}\" yarn run -T ts-node ./scripts/publish-npm.ts",
-    "lint": "yarn run  -T eslint && yarn run -T stylelint",
-    "lint:js": "yarn run -T eslint --cache '**/*.[jt]s?(x)'",
-    "lint:css": "yarn run -T stylelint 'src/**/*.scss'",
+    "lint": "pnpm run lint:js && pnpm run lint:css",
```

Comment:
Here's where I did `pnpm run [script]` vs `pnpm [npm bin]` to treat scripts vs package binds different. I guess I'll drop `run` from all of them?

Reply:
That would be my preference, but I don't feel strongly.